File: src/main/java/com/extendedae_plus/mixin/impl/IOerMEStorage.java
package com.extendedae_plus.mixin.impl;

import appeng.api.config.Actionable;
import appeng.api.networking.security.IActionSource;
import appeng.api.stacks.AEKey;

@FunctionalInterface
public interface IOerMEStorage {
    long apply(AEKey what, long amount, Actionable mode, IActionSource source);
}

File: src/main/kotlin/com/extendedae_plus/mixin/impl/IOerMEStorage.kt
package com.extendedae_plus.mixin.impl

import appeng.api.config.Actionable
import appeng.api.networking.security.IActionSource
import appeng.api.stacks.AEKey

fun interface IOerMEStorage {
    fun apply(what: AEKey?, amount: Long, mode: Actionable?, source: IActionSource?): Long
}
